### PR TITLE
fix package name, refactor constants

### PR DESCRIPTION
**Pull request by sin3point14**

### CODE REVIEW

**Comment by 4Denthusiast** [comment]
Shouldn't "Volcanoes" in the package names also be lowercase?

File: src/main/java/org/Terasology/Volcanoes/VolcanoProvider.java [hunk 1]
```diff
@@ -29,6 +29,7 @@
 })
 @Produces(VolcanoFacet.class)
 public class VolcanoProvider implements FacetProviderPlugin {
+    private static final int DENSITY_SAMPLING_FACTOR = 0.07;
```
Comment (Cervator) [inline src/main/java/org/Terasology/Volcanoes/VolcanoProvider.java]: Thinking you meant `float` here? Easy fix while merging, tests fine as `float` 👍

**Review by Cervator — APPROVED** [review]
Other than a variable type issue I'm pretty sure was _whoopsed_ this tested out as otherwise intended - including having to fix casing in the package _directories_ locally. Have seen issues with case-only changes not sticking, so that could have been done locally for all I know then not committed right :-)

Found one volcano after a few minutes which seems a nice amount of rarity 👍 

Edit: yep, when I went to try and commit this in IntelliJ _or_ via CLI Git on Win10 it didn't catch the directory renames either.

Did a `git config core.ignorecase false` in the Volcanoes directory and then it was caught correctly.

**Comment by Cervator** [comment]
Oh, also went ahead and changed default branch to `develop` and deleted `master`

**Pull request merged**